pubs/plugs/git/git.py

```diff
@@ -0,0 +1,94 @@
+
+import subprocess
+from pipes import quote as shell_quote
+
+from ...plugins import PapersPlugin
+from ...events import *
+
+
+class GitPlugin(PapersPlugin):
+
+    name = 'git'
+    pubsdir = None
+
+    def __init__(self, conf):
+        self.description = "Run git commands in the pubs directory"
+        # Needed for the event listening
+        GitPlugin.pubsdir = conf['main']['pubsdir']
+
+    def update_parser(self, subparsers, conf):
+        git_parser = self.parser(subparsers)
+        git_parser.set_defaults(func=self.command)
+
+    def parser(self, parser):
```

Comment:
Could be merged with the previous one since there is no real difference between both.